.github/CODEOWNERS

```diff
@@ -7,6 +7,10 @@
 docs/compilers @dotnet/roslyn-compiler
 docs/ide @dotnet/roslyn-ide
 
+# Notify docs team for new breaking change documentation
+docs/compilers/CSharp @dotnet/docs
```

Comment:
This should not be in this PR. It will be removed from `main` anyways. Likely related to PR targeting `release/dev17.3` instead of `main`.